docs/root/version_history.md

```diff
@@ -1,6 +1,19 @@
 Version history
 ---------------
 
+0.3 (TBD)
+=========================
+
+### Notable (breaking) changes
+
+- In `service.proto` a change was made to allow both `Output` and `error_detail` to co-exist at the same time.
+- Both `nighthawk_client` and `nighthawk_service` will indicate execution failure (respectively through exit code or grpc reply) when connection errors and/or status code errors are observed by default.
+
+
+### Changelist
+
+- Introducing termination predicates (https://github.com/envoyproxy/nighthawk/pull/167)
```

Comment:
Also would be good to describe the behavioral change.

Reply:
Well, I did create a section with breaking changes which does so, see https://github.com/envoyproxy/nighthawk/pull/167/files#diff-381db20629d045ee9d5e7567c36a451cR10
Given that, WDYT?